containers/docker-compose.yaml

```diff
@@ -136,6 +136,12 @@ services:
       - GDS_BFF_MODE=release
       - GDS_BFF_LOG_LEVEL=debug
       - GDS_BFF_CONSOLE_LOG=true
+      - GDS_BFF_TESTNET_INSECURE=true
+      - GDS_BFF_TESTNET_ENDPOINT=gds:4436
```

Comment:
not 4437?

Reply:
Oop, actually it's supposed to be :4433 - to match the GDS bind addr to connect on; :4436 is trtl, I guess I just got used to typing it. Thanks for catching!